slides/01-introduction.qmd

```diff
@@ -141,3 +141,116 @@ inference vs. prediction
 library(tidymodels)
 ```
 
+##  {background-image="images/tm-org.png" background-size="contain"}
+
+## The whole game
+
+```{r diagram-split, echo = FALSE}
+#| fig-align: "center"
+
+knitr::include_graphics("images/whole-game-split.jpg")
+```
+
+## The whole game
+
+```{r diagram-model-1, echo = FALSE}
+#| fig-align: "center"
+
+knitr::include_graphics("images/whole-game-model-1.jpg")
+```
+
+## The whole game
+
+```{r diagram-model-n, echo = FALSE}
+#| fig-align: "center"
+
+knitr::include_graphics("images/whole-game-model-n.jpg")
+```
+
+## The whole game
+
+```{r diagram-resamples, echo = FALSE}
+#| fig-align: "center"
+
+knitr::include_graphics("images/whole-game-resamples.jpg")
+```
+
+## The whole game
+
+```{r diagram-select, echo = FALSE}
+#| fig-align: "center"
+
+knitr::include_graphics("images/whole-game-select.jpg")
+```
+
+## The whole game
+
+```{r diagram-final-fit, echo = FALSE}
+#| fig-align: "center"
+
+knitr::include_graphics("images/whole-game-final-fit.jpg")
+```
+
+## The whole game
+
+```{r diagram-final-performance, echo = FALSE}
+#| fig-align: "center"
+
+knitr::include_graphics("images/whole-game-final-performance.jpg")
+```
+
+
+## Let's install some packages
+
+If you are using your own laptop instead of RStudio Cloud:
+
+```{r}
+#| eval: false
+install.packages(c("Cubist", "DALEXtra", "doParallel", "earth", "embed", 
+                   "forcats", "lme4", "parallelly", "ranger", "remotes", "rpart", 
+                   "rpart.plot", "rules", "stacks", "tidymodels",
+                   "vetiver", "xgboost"))
+
+remotes::install_github("topepo/ongoal@hockeyR")
+```
+
+. . .
+
+<br></br>
+
+### <i class="fa fa-cloud"></i> Or log in to RStudio Cloud:
+
+[bit.ly/tidymodels-iceland-2022](http://bit.ly/tidymodels-iceland-2022)
```

Comment:
I will make a cloud project tomorrow and will update this link.

Reply:
I've changed the link in the slides to `http://bit.ly/tidymodels-nyr-2023`. If you make the bitly link point to the new clour project, we should be set!